goworld.ini.sample

```diff
@@ -2,7 +2,7 @@
 debug = 1 ; set to 0 in production
 
 [deployment]
-desired_dispatchers=1
+desired_dispatchers=2
```

Comment:
kindly suggestion: do not involve unrelated change

Reply:
I didn't give this specific line much thought. It's just there are more than one dispatchers in the sample. And it doesn't seem like affecting much on dev phases by increasing on the count. You can reject this if you don't like it.

From my point of view, the sample config should be verbose about more than one dispatcher. And since all actual run time config were created by copying this sample file. It would be a good choice to default the count of dispatchers to at least 2.